### PR TITLE
Refactor publish/release graphs

### PR DESCRIPTION
**Pull request by cjllanwarne**
- Combines the hotfix and regular release graphs into a single diagram
- Removes the redundant "re-run swatomation" step from the end of the release process
- Add the creation of a new "work in progress version" PR to firecloud-develop

Rendered Image:

### CODE REVIEW

File: scripts/release_processes/README.MD [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Release Processes
 
+## How to update these processes
+
+Have a better idea about how the deployment process should go? Make a PR and it'll be reviewed, just like a code change!
+
+ * Navigate to this directory
+ * Modify the appropriate `.dot` file 
+ * Run `refresh.sh` to update the png files.
```
Comment (aednichols) [inline scripts/release_processes/README.MD]: I do not find any file by this name in this PR nor in the repo on `develop`

File: scripts/release_processes/firecloud-develop.dot [hunk 2]
```diff
@@ -2,40 +2,71 @@ digraph {
...
+    confirm_cromwell_v [shape=oval label="[Jenkins] Check for the correct Cromwell version hash in fiab-start logs\n(otherwise re-run 'dsl-seed' and use 'jenkins retest')"];
+    confirm_cromwell_v [shape=oval label="[Jenkins] Check for the correct Cromwell version hash in fiab-start logs\n(otherwise re-run 'dsl-seed' and use 'jenkins retest')"];
+    fcdev_success [shape=oval label="Wait for swatomation to succeed"];
+
+    fcdev_merge [shape=oval label="Merge firecloud-develop PR"];
+
+    dspjenkins_PR [shape=oval label="Make dsp-jenkins PR setting new Cromwell version"];
+    dspjenkins_merge [shape=oval label="Merge dsp-jenkins PR"];
+
+    qa_perf [shape=oval label="QA performance testing (skip for hotfix releases)"];
```
Comment (andy7i) [inline scripts/release_processes/firecloud-develop.dot]: Suggestion:
`(for hotfix releases, this step is not blocking, but these tests can be run on the side)`

File: scripts/release_processes/README.MD (hunk 1 above)
Comment (cjllanwarne) [reply scripts/release_processes/README.MD]: Oops, forgot to add the file. Should be there now.

File: scripts/release_processes/refresh.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```
Comment (kshakir) [inline scripts/release_processes/refresh.sh]: ToL: Possibly akin to the "Don't use `Some.apply()`!" recommendations.

```suggestion
#!/usr/bin/env bash
```

https://unix.stackexchange.com/questions/206350/what-is-the-difference-if-i-start-bash-with-bin-bash-or-usr-bin-env-bash

**Pull request merged**